schemas/rule.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: rule
+  title: Rule
+  group: 2
+  short: Fields to capture details about rules used to generate alerts or other notable events.
+  description: >
+   Rule fields are used to capture the specifics of any observer or agent rules that generate alerts or other notable events.
+
+   Examples of data sources that would populate the rule fields include: network admission control platforms, network or 
+   host IDS/IPS, network firewalls, web application firewalls, url filters, endpoint detection and response (EDR) systems, etc.  
+  
+  type: group
+  fields:
+
+    - name: id
+      level: extended
+      type: keyword
+      short: Rule ID
+      description: >
+        A rule ID that is unique within the scope of an agent, observer, or other entity using the rule for detection of this event.
+
+      example: 101 
+
+    - name: uuid
+      level: extended
+      type: keyword
+      short: Rule UUID
+      description: >
+        A rule ID that is unique within the scope of a set or group of agents, observers, or other entities using the rule for detection of this event.
+
+      example: 1100110011
+
+    - name: version
+      level: extended
+      type: keyword
+      short: Rule version
+      description: The version / revision of the rule being used for analysis.
+      example: 1.1
+
+    - name: name
+      level: extended
+      type: keyword
+      short: Rule name
+      description: The name of the rule or signature generating the event.
+      example: BLOCK_DNS_over_TLS
+
+    - name: description
+      level: extended
+      type: keyword
+      short: Rule description
+      description: The description of the rule generating the event.
+      example: Block requests to public DNS over HTTPS / TLS protocols
```

Comment:
I'm assuming the plan was to do a subsequent PR with the corresponding #680 changes that likely ought to be added?

Reply:
Yeah the default text indexer would be useful here, but we should do another round later.